### ISSUE TITLE
Review grml.org/qemu

### ISSUE THREAD

**Issue by jkirk** [grml/grml.org#71]
https://grml.org/qemu/ describes how to use Grml with QEMU.

It seems out of date and should be updated.

It is also hard to find. Maybe we should create a new Documentation section or mention it in the [documentation](https://grml.org/docs/) page.

**Comment by mika** [comment]
Yes, I'm very unhappy with the doc state as-is, also that we need to manually integrate the resulting documentation files from grml2usb, grml-debootstrap etc in the grml.org git repository, tending to be annoying and easily forgettable, and therefore often outdated. :(

**Comment by zeha** [comment]
Do we want to do more here?

**Comment by mika** [comment]
I think the current state thanks to your (@zeha) https://github.com/grml/grml.org/pull/80/changes/8a9cf996dc78afea7014368aa65611ddea5cd516 is "good enough", don't think we want to remove it, so let's consider this as resolved for now?
